Doc/library/http.server.rst

```diff
@@ -438,10 +438,10 @@ to bind to localhost only::
 
         python -m http.server --bind 127.0.0.1
 
-.. versionadded:: 3.4
+.. versionchanged:: 3.4
     ``--bind`` argument was introduced.
```

Comment:
```suggestion
    Added the ``--bind`` option.
```

Use clear, consistent grammar and phrasing like the others, and active voice.

Ideally the arguments would be documented as actual ``.. option`` and ``...versionadded`` and the rest used underneath them, but perhaps a little too much to ask here (wouldn't be that hard though)?

Reply:
IMO, it is fine to such amendments in this PR.